src/App.css

```diff
@@ -1,26 +1,49 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab&display=swap');
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
 }
 
+body {
+  background-color: #fafafa;
+}
+
+/*  Navbar */
 .navBar {
-  height: 4rem;
   display: flex;
-  align-items: center;
-  padding: 0 2rem;
+  flex-direction: row;
   justify-content: space-between;
+  position: relative;
+  align-items: center;
+  font-family: 'Montserrat', sans-serif;
+  padding: 1% 5%;
+  background-color: #fff;
+  width: 100%;
+}
```

Comment:
- [ ] Please 🙏🏽 align the navbar and its content as described to be similar to the Zeplin design 

| Zeplin design | Your design |
|---|---|
| ![image](https://github.com/Javed-Shahbaz/book-store-react/assets/58642949/0fc98dd4-5788-4429-a07a-36a6f66ad06f) | ![image](https://github.com/Javed-Shahbaz/book-store-react/assets/58642949/bba62f6b-0340-4c4b-9167-e2728f28b833) |